mlflow/utils/string_utils.py

```diff
@@ -14,6 +14,13 @@ def is_string_type(item):
     return isinstance(item, str)
 
 
+def stringify_column(s):
+    if isinstance(s, str):
+        return s
+
+    return f"feature_{s}"
```

Comment:
I suggest to use `return f"feature_{s}_{uuid.uuid4()}"` to avoid generate duplicate feature names.

Reply:
A pandas dataframe normally contains duplicate column names. Do we reaaly need to add uuid?

Reply:
I think `uuid.uuid4().hex[:6]` confuses users. If I saw `feature_0_af82jk`, I would wonder what `af82jk` is and where it comes from.

Reply:
Let me know if I'm missing something.

Reply:
I agree that a pandas dataframe is unlikely to include duplicate column names. If the column names are uninitialized, they will automatically be named with integers anyway. I'll leave off the uuid for now unless you really want it @WeichenXu123.

Reply:
OK I agree. let's keep current code